test/e2e/flipcard.cy.js

```diff
@@ -33,16 +32,16 @@ describe('Flipcard', function () {
           cy.get('.flipcard__item-frontImage')
             .should('have.attr', 'src', frontImage.src)
             .should('have.attr', 'aria-label', frontImage.alt);
-
+          
           // Make sure interacting switches the front and back correctly
           cy.get('.flipcard__item-front').should('be.visible');
           cy.get('.flipcard__item-back').should('not.be.visible');
-
-          cy.get('.flipcard__item-front').click({force: true});
```

Comment:
Is this no longer needed?

Reply:
Accidentally lost through merging so adding back now